src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationResource.kt

```diff
@@ -24,11 +24,11 @@ import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.Migration
 @RestController
 @RequestMapping("/migrate/contactperson", produces = [MediaType.APPLICATION_JSON_VALUE])
 @Tag(name = "contact-person-migration-resource")
+@PreAuthorize("hasAnyRole('ROLE_MIGRATE_CONTACTPERSON', 'ROLE_MIGRATE_NOMIS_SYSCON')")
```

Comment:
New generic role for migrations that sycon is likely to run. This means we will need need to keep on asking for new roles if we know the migration will be run by syscon